### ISSUE TITLE
Mark search matches with special color (UI/UX)

### ISSUE THREAD

**Issue by n0tmyproblem** [google/timesketch#1780]
**Is your feature request related to a problem? Please describe.**
In the explore tab, when a search about specific fields content or global content generate any matches, it could require to spend some seconds (or Ctrl+F) searching the places where matches have been found.

**Describe the solution you'd like**
I think it could be interesting to mark in different color the search matches, to improve the eyes detection and speed review.

**Describe alternatives you've considered**
Any

**Additional context**
Any

Thanks,
- NMP

**Comment by berggren** [comment]
Yeah, I agree that would be great. I have been thinking about this as well. I'll keep this issue to track this FR.

Thanks!

**Comment by jaegeral** [comment]
@tomchop who has been working on something similar lately, WDYT?

**Comment by tomchop** [comment]
I think it would be a good idea to highlight the table row in which the query matches! I'm just slightly worried about more complex queries that would involve some parsing before deciding what to highlight (e.g. `label:foo`, `foo AND bar`)

**Comment by josephsam-hub** [comment]
hi,  i am interested to update this future , let me.

**Comment by jkppr** [comment]
Hi @josephsam-hub let's focus on your open PRs first. We'll look into other issues you can tackle later once they are merged.

**Comment by josephsam-hub** [comment]
ok fine . @jkppr let go with the old PR.
